oap-server-bom/pom.xml

```diff
@@ -72,7 +72,7 @@
         <awaitility.version>3.0.0</awaitility.version>
         <httpcore.version>4.4.13</httpcore.version>
         <commons-compress.version>1.21</commons-compress.version>
-        <banyandb-java-client.version>0.4.0-SNAPSHOT</banyandb-java-client.version>
+        <banyandb-java-client.version>0.4.0-rc0</banyandb-java-client.version>
```

Comment:
Is the rc0 already out on maven central?

Reply:
> Is the rc0 already out on maven central?

Yes. I've released last night.

Reply:
Please make sure you sent the mail list update and other process first.